src/home_robot/home_robot/control/goto_controller.py

```diff
@@ -123,6 +123,6 @@ def compute_control(self):
         xyt_err = self._compute_error_pose()
 
         # Compute control
-        v_cmd, w_cmd = self.control(xyt_err)
+        v_cmd, w_cmd, done = self.control(xyt_err)
 
-        return v_cmd, w_cmd
```

Comment:
Actually, can we implement an `is_done()` function and check that within the ROS node?
Changing this output will break waypoint control in Habitat-lab, which is currently using `GotoVelocityController`.

Reply:
updated this